components/massmint/types.ts

```diff
@@ -16,15 +16,15 @@ export type NFT = {
   description?: string
   price?: number
   status?: Status
-  attributes?: OpenSeaAttribute[]
+  tags?: OpenSeaAttribute[]
```

Comment:
why the change to `tags` I'd keep it as `attributes`

Reply:
same reason as below comment